components/card/README.md

```diff
@@ -47,79 +154,107 @@ The `d2l-card` element is a container that provides specific layout using severa
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
   </div>
 </d2l-card>
 ```
 
-**Properties:**
-
-| Property | Type | Description |
-|--|--|--|
-| `icon` | String, required | Preset icon key (ex. "tier1:gear") |
-| `text` | String, required | Accessible text for the link (not visible, gets announced when user focuses) |
-| `download` | Boolean | Download a URL instead of navigating to it |
-| `href` | String | Location for the primary action/navigation |
-| `rel` | String | Relationship of the target object to the link object |
-| `secondary-text` | String | Text to display as a superscript on the icon |
-| `secondary-text-type` | String | Controls the style of the secondary text bubble; options are `notification` and `count` |
-| `target` | String | Where to display the linked URL |
-
-See the [anchor element docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) for more information on standard link attributes and their values.
-
-## d2l-card-content-title
+## Card content: Meta  <d2l-card-content-meta>
 
-The `d2l-card-content-title` element is a helper for providing layout/style for a title within the `content` slot.
+The `d2l-card-content-meta` element is a helper for providing layout/style for a meta data within the `content` slot.
 
+<!-- docs: demo live name:d2l-card-content-meta size:large -->
 ```html
 <script type="module">
   import '@brightspace-ui/core/components/card/card.js';
-  import '@brightspace-ui/core/components/card/card-content-title.js';
+  import '@brightspace-ui/core/components/card/card-content-meta.js';
+  import '@brightspace-ui/core/components/card/card-footer-link.js';
+  import '@brightspace-ui/core/components/tooltip/tooltip.js';
 </script>
-
-<d2l-card>
+<d2l-card align-center text="Biology" align-center href="#" style="height: 320px; width: 245px;">
+  <img slot="header" alt="" style="display: block; width: 100%;" src="https://s.brightspace.com/course-images/images/7905e442-f009-46f6-8586-2c273a7c0158/banner-narrow-low-density-max-size.jpg">
   <div slot="content">
-    <d2l-card-content-title>Hydrology</d2l-card-content-title>
+    <div>Biology</div>
+    <d2l-card-content-meta>Science • Grade 11</d2l-card-content-meta>
+  </div>
+  <div slot="footer">
+    <d2l-card-footer-link id="discussionsLink1" icon="tier1:outcomes" text="Outcomes" secondary-text="2"></d2l-card-footer-link>
+    <d2l-tooltip for="discussionsLink1">Discussions</d2l-tooltip>
+    <d2l-card-footer-link id="assignmentsLink1" icon="tier1:assignments" text="Assignments" secondary-text="1"></d2l-card-footer-link>
+    <d2l-tooltip position="top" style="width: 100%;" for="assignmentsLink1">You have 1 assignments due tomorrow.</d2l-tooltip>
   </div>
 </d2l-card>
 ```
 
-## d2l-card-content-meta
+## Card Footer: Link <d2l-card-footer-link>
```

Comment:
This one can probably just be `Card Footer Link` - can just see what Jeff thinks on review too